Source/WebCore/html/HTMLFormControlElement.cpp

```diff
@@ -686,4 +294,37 @@ bool HTMLFormControlElement::needsMouseFocusableQuirk() const
     return document().quirks().needsFormControlToBeMouseFocusable();
 }
 
+void HTMLFormControlElement::didFinishInsertingNode()
```

Comment:
Can we put this after HTMLFormControlElement::insertedIntoAncestor below?

Reply:
Reordered everywhere this PR touched those methods.